mm2src/coins/nft.rs

```diff
@@ -648,22 +702,28 @@ async fn get_moralis_nft_list(
 pub(crate) async fn get_nfts_for_activation(
     chain: &Chain,
     my_address: &Address,
-    url: &Url,
+    original_url: &Url,
+    signed_message: &GuiAuthValidation,
```

Comment:
This as well.

Reply:
The `get_nfts_for_activation` function is only used for NFT activation. Since it doesn't require an antispam URL, I don't think we should create a wrapper with a dummy Url value.
![Screenshot 2024-05-10 at 15 11 18](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/a130adf4-b3c3-49da-978a-9d1d1cf4f1bc)